docs/source/upgrade-guide/index.md

```diff
@@ -265,6 +265,37 @@ It requires the latest version of `plone.restapi` (>=8.42.0) installed in the ba
 The `utils.js` file of the Teaser block was removed because it is no longer used.
 You can consider removing it if you were shadowing it in your project.
 
+(grid-block-migration-from-kitconcept-volto-blocks-grid-label)=
+
+### Grid block migration from `@kitconcept/volto-blocks-grid`
+
+The grid block was added to Volto in version 17.0.0-alpha.16.
+It is based on the `@kitconcept/volto-blocks-grid` add-on version 7.x.x.
+
+If you previously used `@kitconcept/volto-blocks-grid` in your project based on a Volto version before 17.0.0-alpha.16, then your site will show two available block types both named `Grid`.
```

Comment:
@sneridagh @davisagli can you verify this sentence for accuracy?

Reply:
It is accurate, then later it explains how to deal with that situation, by using `restricted` property.

Reply:
Merging!